pymc3/tests/test_distributions_random.py

```diff
@@ -969,7 +991,6 @@ def ref_rand(size, mu, lam, alpha):
             ref_rand=ref_rand,
         )
 
-    @pytest.mark.xfail(reason="This distribution has not been refactored for v4")
```

Comment:
If this passes locally, you can go ahead and delete it as well (do test it before!)

Reply:
I did try to test on my local, but the following error occured

```
ImportError while loading conftest '/Users/farhanreynaldo/Documents/pymc3/pymc3/tests/conftest.py'.
pymc3/__init__.py:41: in <module>
    from pymc3 import gp, ode, sampling
pymc3/gp/__init__.py:16: in <module>
    from pymc3.gp.gp import TP, Latent, LatentKron, Marginal, MarginalKron, MarginalSparse
pymc3/gp/gp.py:35: in <module>
    from pymc3.math import (
pymc3/math.py:31: in <module>
    from aesara.tensor import (
E   ImportError: cannot import name 'sigmoid' from 'aesara.tensor' (/Users/farhanreynaldo/miniconda3/envs/pymc3-dev-py39/lib/python3.9/site-packages/aesara/tensor/__init__.py)
```

Reply:
You need to update aesara to V2.0.8